### DISCUSSION TITLE
Blog "Software Architecture Document (SAD)" - PeerReviews

### DISCUSSION THREAD

**Discussion by open-schnick** [FileFighter/filefighter.github.io#34] [General]
Leave your feedback about our new blog post below :D

**Comment by nilswendland** [comment]
Hey FileFighters,
it looks like you already have some experience in using your IDE to create such documents. They all look great. If you also know how to do these with Android studio I would love to get your insight on this.

Your SAD states the following under Section 3: The script will initialize all necessary services, start the FileFighter nas and also periodical check for new versions of the services.

I am unclear what "nas" stands for in this context. Maybe it is an abbreviation, in that case I would add it to the list, else I suggest you elaborate a little more, so someone outside the project understands what is meant by it.

Keep up the good work

Your Team Sweat4Success

**Comment by open-schnick** [comment]
> Hey FileFighters,
> it looks like you already have some experience in using your IDE to create such documents. They all look great. If you also know how to do these with Android studio I would love to get your insight on this.
> 
> Your SAD states the following under Section 3: The script will initialize all necessary services, start the FileFighter nas and also periodical check for new versions of the services.
> 
> I am unclear what "nas" stands for in this context. Maybe it is an abbreviation, in that case I would add it to the list, else I suggest you elaborate a little more, so someone outside the project understands what is meant by it.
> 
> Keep up the good work
> 
> Your Team Sweat4Success

Thanks, we are definitely going to implement a definition of this abbreviation in the future.

**Comment by DSinMeliodas** [comment]
Dear FileFighter Team,
first of all I want to mention that your SAD is very well structured just like all what you have done until now.  The pictures you use in your SAD are also quiet good and support your explanations very well.

Like the previous review I also want to point out a missing abbreviation:
CI/CD Section 3, 2nd Paragraph: "That way we can manage Cl/CD, pipelines ..."

Another point that I want to mention is your full class diagram even you can click on it to make it readable you lose track of the structure itself so I would recommend you to try to splitt it up in order to make it better readable and understandable.

with best regards
the Futurum Team
